tests/stub/authorization.py

```diff
@@ -0,0 +1,634 @@
+from nutkit.frontend import Driver
+import nutkit.protocol as types
+from tests.shared import (
+    get_driver_name,
+    TestkitTestCase,
+)
+from tests.stub.shared import StubServer
+
+
+def get_extra_hello_props():
+    if get_driver_name() in ["java"]:
+        return ', "realm": ""'
+    elif get_driver_name() in ["javascript"]:
+        return ', "realm": "", "ticket": ""'
+    return ""
+
+
+class Tests(TestkitTestCase):
```

Comment:
Maybe give it a more descriptive name like `AuthorizationTests` or such.

Reply:
Good point, I have updated the PR, thank you 👍